messaging/src/test/java/com/iluwatar/messaging/MessagingApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package com.iluwatar.messaging;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class MessagingApplicationTests {
+
+  @Test
+  void contextLoads() {
+  }
```

Comment:
Is it purposefully empty? Add comments to explain if so.

Reply:
updated